### PR TITLE
Update installation notes to readme

### PR DESCRIPTION
**Pull request by jazzsequence**
This adds updated installation notes including how to install without integrated composer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +21,41 @@ composer require pantheon-systems/edge-integrations-wordpress-sdk
...
+If you aren't running a project with Integrated Composer but you _do_ use Composer otherwise, you can still get started with the Edge Integrations WordPress SDK without too much trouble. This still assumes you have Composer installed on at least one machine.
+
+#### Step 1: Clone the WordPress Edge Integrations plugin
+
+First, you'll need to get a copy of the [WordPress Edge Integrations plugin](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations). Clone or download the plugin into your `wp-content/plugins` directory.
+
+```bash
+git clone git@github.com:pantheon-systems/pantheon-wordpress-edge-integrations.git
+```
+
```
Comment (jspellman814) [inline README.md:40]: Not sure if it's worth including but the plugin can also be pulled from packagist using
`composer require pantheon-systems/pantheon-wordpress-edge-integrations`

File: README.md [hunk 2]
```diff
@@ -19,3 +21,41 @@ composer require pantheon-systems/edge-integrations-wordpress-sdk
...
+#### Step 2: Install Pantheon Edge Integrations library
+
+Once you have the plugin downloaded, you'll need to run `composer install` to get the Pantheon Edge Integrations library. `cd` into the plugin directory and run `composer install`. From the `wp-content/plugins` directory, run the following commands:
```
Comment (jspellman814) [inline README.md]:
```suggestion
Once you have the plugin downloaded, you'll need to run `composer install` to get the Pantheon Edge Integrations library. `cd` into the plugin's directory and run `composer install`. From the `wp-content/plugins` directory, run the following commands:
```

**Review by jspellman814 — APPROVED** [review]
Couple small comments that shouldn't be a blocker. Overall looks great!

File: README.md (hunk 1 above)
Comment (jazzsequence) [reply README.md:40]: I suppose I could add it as a note since this still assumes that Composer is installed on the client machine.

**Pull request merged**